<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basketeditDlg</class>
 <widget class="QDialog" name="basketeditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select field to edit ...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="editBasket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
